components/menu/stories/menu.stories.js

```diff
@@ -42,18 +51,38 @@ export default {
 			control: "text",
 			if: { arg: "shouldTruncate", truthy: true },
 		},
-		hasActions: { table: { disable: true } },
+		hasActions: {
```

Comment:
I didn't see the "Has switches" having an effect in the "Menu With Variants" in multiple selection.

Reply:
This control is only visible for single menu item now!